docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml

```diff
@@ -10,7 +10,18 @@ spring:
       reply-timeout: 7200000
   cloud:
     stream:
+      function:
+        definition: interrupt;request
```

Comment:
As this must not be modified, maybe we can remove it from "user" config

Reply:
Yes it can be. In fact the jar contains his configuration and the yml override it by mounting it in the docker. By reflex I still put a complete configuration, I don't like hiding elements even if they are static.

I will remove these two lines